extension_config.cmake

```diff
@@ -7,7 +7,7 @@ duckdb_extension_load(aws
 )
 
 duckdb_extension_load(httpfs
-        GIT_URL https://github.com/duckdb/duckdb-httpfs
-        GIT_TAG 354d3f436a33f80f03a74419e76eb59459e19168
-        INCLUDE_DIR extension/httpfs/include
+    GIT_URL https://github.com/duckdb/duckdb-httpfs
```

Comment:
I thought you changed this to using https://github.com/duckdb/duckdb-delta/blob/03aaf0f073bc622ade27c158d32473588b32aa8b/Makefile#L23 ?

Reply:
see #118